backend/src/database/repositories/userRepository.ts

```diff
@@ -186,7 +200,7 @@ export default class UserRepository {
       data.jwtTokenInvalidBefore = new Date()
     }
 
-    await user.update(data, { transaction })
+    await this.update(user.id, data, options)
```

Comment:
you don't have to call this.update you can just bust cache at the user.update call.